app/controllers/users/verify_password_controller.rb

```diff
@@ -6,14 +6,19 @@ class VerifyPasswordController < ApplicationController
     before_action :confirm_password_reset_profile
     before_action :confirm_personal_key
 
-    def new; end
+    def new
+      analytics.track_event(:reactivate_account_verify_password_visited)
```

Comment:
wouldn't this just be `analytics.reactivate_account_verify_password_visited` ?

Reply:
lol, yep